src/commands/VerifyCommand.ts

```diff
@@ -29,10 +29,18 @@ export class VerifyCommand implements ICommand {
     constructor(private readonly client: MatrixClient, private readonly conference: Conference) {}
 
     public async run(roomId: string, event: any, args: string[]) {
-        const audId = args[0];
+        let audId;
+        let backstage = args.includes("backstage")
```

Comment:
I guess we probably need to do:
```suggestion
        let backstage = args[args.length - 1] === "backstage";
```